storage/tianmu/system/configuration.h

```diff
@@ -88,6 +88,12 @@ extern char tianmu_sysvar_enable_histogram_cmap_bloom;
 extern unsigned int tianmu_sysvar_result_sender_rows;
 // Number of threads executing (delete_row) and (update_row) in parallel
 extern unsigned int tianmu_sysvar_delete_or_update_threads;
+// The parallelism degree of multithreaded aggregation
+// the number of threads executing group by multithreaded aggregation
+extern unsigned int tianmu_sysvar_groupby_parallel_degree;
+// Threshold for the minimum number of rows
+// that can start executing a multithreaded group by thread
+extern unsigned long long tianmu_sysvar_groupby_parallel_rows_minimum;
```

Comment:
I dont think that taking the number of rows as a threshold is suite choice.  io throughput is the most important factor which impact the performance. the io thoughput has some factors: one is the row we scan. and another one is row size.  Here, you just are taking one thing under considering. and how about we just only one column and which type is bit(1). Here, Do we really need to use parallel aggregation?

Reply:
This value does have to be debated, but setting the number of rows is the simplest and most straightforward for the user to understand